Applications/MaterialManager/Contracts/Processing/Optimization/OffcutParameterSet.cs

```diff
@@ -14,13 +14,16 @@ public class OffcutParameterSet : IValidatableObject, IContainsUnitSystemDepende
     private const int _MaterialGroupNameMaxLength = 50;
 
     /// <summary>
-    /// Gets or sets the name of the material group.
+    /// Gets or sets the <see cref="IsDefault" /> which determines if the parameter set belongs to the default section.
```

Comment:
Hi, I dont like the working belongs to the default section because that is really MM specific and does not say much I think. 
May be something like: IsUnassignedMaterialsGroup would be better and this comment would say "if the group contains all materials which are not manually assigned to a group." or something like that. I would like to have this bool speak for itself incase customers of HG connect use this model

Reply:
Fine for me. I will adjust it.